sysml.library/Kernel Libraries/Kernel Function Library/BaseFunctions.kerml

```diff
@@ -45,25 +45,25 @@ standard library package BaseFunctions {
     
 	abstract function 'istype'{ 
 		in seq: Anything[0..*];
-		abstract feature 'type': Anything;
+		in 'type': Anything;
 		return : Boolean[1];
 	}
 	
 	abstract function 'hastype'{ 
 		in seq: Anything[0..*];
-		abstract feature 'type': Anything;
+		in 'type': Anything;
 		return : Boolean;
```

Comment:
For consistency with other functions in package BaseFunctions type should be `Boolean[1]`, i.e. add explicit multiplicity.
Aternative remove `[1]` (which is the default multiplicity) from the other three functions.